src/main/java/org/opentripplanner/routing/impl/DefaultFareServiceImpl.java

```diff
@@ -159,6 +159,8 @@ public void addFareRules(FareType fareType, Collection<FareRuleSet> fareRules) {
         fareRulesPerType.put(fareType, new ArrayList<>(fareRules));
     }
 
+    // TODO OTP2 Removing transit search from A* breaks this. I have commented out some code to
+    //      get it to compile.
```

Comment:
We need to come up with a new approach to calculating fares then. I guess we need to construct the Ride objects from the raptor results.

Reply:
Resolved this problem in commit c4ef215

Reply:
When I tested Raptor in r5 with the SpeedTest I got a significant time penalty if I tryed to map all paths to itineraries. So, we should probably have support for decorating/filtering services of the results both on paths and itineraries. Any FareService should just implement such generic filter/decorator. This is out-out-scope for this PR, but I think it is worth having in the back of the head while doing this PR.